spec/models/profile_spec.rb

```diff
@@ -0,0 +1,45 @@
+ require 'spec_helper'
+
+ describe Profile do
```

Comment:
Hey @essymo, I'm currently trying to find out how to make these tests passing. One thing is that an email address can only be used once. So if you create a profile out of the factory you would create two contexts and move the let block separately into each context (so it is only valid in this specific context and the email address can be used in the next context again without problems). See how I refactored the code - now the tests pass: https://gist.github.com/zaziemo/8df5b517a04fac0fe1cd